spec/rspec/core/memoized_helpers_spec.rb

```diff
@@ -520,12 +520,39 @@ def count
       end.to raise_error(/#let or #subject called without a block/)
     end
 
-    it 'raises an error when attempting to define a reserved method name' do
-      expect do
-        RSpec.describe { let(:initialize) { true }}
-      end.to raise_error(/#let or #subject called with a reserved name #initialize/)
+    context 'when defining a reserved name' do
+      context 'when name is initialize' do
+
+        it 'raises an error when attempting to define a reserved name #initialize' do
+          expect do
+            RSpec.describe { let(:initialize) { true }}
+          end.to raise_error(/#let or #subject called with reserved name #initialize/)
+        end
+
+        it 'raises an error when attempting to define a reserved name #initialize as a string' do
+          expect do
+            RSpec.describe { let('initialize') { true }}
+          end.to raise_error(/#let or #subject called with reserved name #initialize/)
+        end
+      end
+
+      context 'when name is to_s' do
```

Comment:
There's really no context for those `'when name is initialize'`/`'when name is to_s'` `context`s. Example descriptions describe it well enough.